source/app/service-providers/fsal/fsal-attachment.ts

```diff
@@ -16,8 +16,9 @@ import path from 'path'
 import { promises as fs } from 'fs'
 import type { OtherFileDescriptor } from '@dts/common/fsal'
 import { shell } from 'electron'
+import type FSALCache from './fsal-cache'
 
-export async function parse (absPath: string): Promise<OtherFileDescriptor> {
+export async function parse (absPath: string, cache: FSALCache): Promise<OtherFileDescriptor> {
```

Comment:
Attachments can now be cached. It's not much, but it makes retrieval of the cached descriptors a thousand times faster. (Went from about 15s to load the entire workspace tree to ca. 8s b/c the files don't have to be fetched from disk anymore.)